.env-transcribe-sample

```diff
@@ -17,6 +17,8 @@ HTR_CLI_IMAGES_FOLDER=/home/user/joplin/packages/transcribe/images
 QUEUE_DRIVER=pg
 # QUEUE_DRIVER=sqlite
 
+FILE_STORAGE_MAINTENANCE_INTERVAL=30000
```

Comment:
A 30 seconds interval seem quite short - any reason why you chose this interval?

Reply:
Ok let's change it to once an hour

Reply:
I don't get what's going on. You changed it everywhere... but to once a minute?

Reply:
Sorry, I fixed to the correct time now.